### ISSUE TITLE
Different answer each time?

### ISSUE THREAD

**Issue by kwalcock** [stanfordnlp/GloVe#207]
Please excuse the naïve question, but should glove.c be creating a different collection of vectors on two runs with the same input even if the random seed also stays the same?  It will do this if the thread count is not 1.  If that's expected behavior, I'm going to have to wait out that single thread in order to get a repeatable result and I'm impatient.  Thanks for any insight anyone can provide.

**Comment by AngledLuffa** [comment]
Yes, a multithreaded process may have the results come in at slightly
different times, leading to math operations being applied in different
orders, leading to small differences which spiral out of control until what
originally looked like a tiny disagreement ending up with a fight lasting
for hours with no one willing to back down over something stupid

**Comment by kwalcock** [comment]
That is explained so eloquently that it makes up for the disappointment.  Thanks for confirming.  I'll just have to wait out the single thread.
